parser/parser_test.go

```diff
@@ -21,6 +21,8 @@ import (
 	"testing"
 
 	"github.com/pingcap/errors"
+	"github.com/stretchr/testify/require"
+
```

Comment:
。。。

Reply:
what's the rule of separating 3-rd party imports? parser/charset/charset.go did so but this file didn't

Reply:
We don't care about it.